change/@ni-nimble-components-4411dde6-cf36-470b-9bfe-f527c5d48d1f.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@m-akinc  before resetting me on PRs
- please make sure the build passes
- please make sure new code follows normal expectations, ie. testing
it would really help reduce the number of re-reviews of PRs and save us all time

Reply:
Ok, since build failures are usually something minor (e.g. lint issue, test needing update, or trivial logic error), I don't expect them to be relevant to reviewer feedback. But I'll try to remember to wait for a green check before resetting you.

> please make sure new code follows normal expectations, ie. testing

Do you want me to add tests for the combobox and mention listbox, or are you talking about something else? I didn't think new tests were necessary for those, but I can add some if you want.

Reply:
> I didn't think new tests were necessary for those

It's not clear to me why if it was valuable for select it was not valuable for other controls doing the same thing.

Reply:
When I wrote the select test, my workaround had been to call `scrollIntoView` right after `focus` instead of delayed with `requestAnimationFrame`. The test was to ensure that scrolling still happened when calling those two functions back-to-back (because FAST's comment suggested they needed to be called in separate frames). Now that I've changed the approach to just short-circuting the focusing and scrolling when `!open`, my original motivation for the test is gone. The new approach should not affect the behavior when opening the listbox, so I didn't think I needed to test that. But there's no harm in having the tests, so I've added them.

Reply:
Gotcha, and what I'm looking for is consistency and avoiding one offs. So when I see a test only testing one code path where we have modified similar code paths (in this case three slightly different ways), I'm wondering why one is tested and the others aren't. So saying it was an unnecessary holdover might mean needing to pause and make another self check of the PR before resetting reviewers. In that scenario before resetting the test may have been removed.

If the test was removed then in that case if you showed and discussed in PR testing section that you found existing tests covering the open + focus behavior in fast foundation and we were calling the super behavior in all three cases and that are changes were minimal, it could make sense to skip adding extra tests. In this case we are only calling the base method in 2 of 3 places so I'd either ask to align (update the mention listbox to also call super) and then could agree that test coverage is probably good enough. Alternatively, in this case we are not aligned so having the test coverage is probably the right call. (And I also didn't check listbox test coverage in fast foundation so don't even know if we'd feel confident in the coverage).

Anyway the above reply isn't requiring an action or anything. Just trying to share the point of view I'm coming from to make our review cycles more efficient.